TUM-Dev/NavigaTUM
# [maps-assignent-2] making the maps-assignet typeing compliant

First PR in the debugging/refactoring effort of https://github.com/TUM-Dev/NavigaTUM/issues/512

## Proposed Changes (include Screenshots if possible)

- refactored the id-change (f.ex. `9`->`rf9`) happens in the scraper
- made sure that the processing interface is fully typed

## How to test this PR

1. run the tests
2. see if it compiles
3. check if the outputted file is identical.

## How has this been tested?

- see above

## Checklist
